content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -92,15 +93,15 @@ To process your project's assets, do one of the following:
 
 - **Run Asset Processor.** Do this if you already built Asset Processor and prefer to use a GUI.
 
-    1. Run `AssetProcessor.exe` from the `<engine>/build/windows/bin/profile` directory.
+    1. Run `AssetProcessor.exe` from the `<project>/build/windows/bin/profile` directory.
 
         {{< image-width "/images/user-guide/packaging/windows-release-build/asset-processor.png" "700" "An image of O3DE Asset Processor." >}}
 
     1. Wait for all jobs to complete. You can view the status of each asset in the **Asset Status** pane and the details of each asset in the **Event Log Details** pane.
 
 - **Run Asset Processor Batch.** Do this if you already built Asset Processor and prefer to use a CLI.
 
-    1. Run `AssetProcessorBatch.exe` from the `<engine>\build\windows\bin\profile` directory.
+    1. Run `AssetProcessorBatch.exe` from the `<project>\build\windows\bin\profile` directory.
```

Comment:
Uh.. about this.
If the user is using a SDK engine, then there Asset Processor would be in the SDK layout directory as part of the engine.

The engine tools like the Asset Processor is placed in the build directory of the project when a project-centric source-engine workflow is used.
When a project-centric SDK engine workflow is used, the AssetProcessor(Batch) has already been built and the one in the SDK layout folder is what gets used.

We should mention the differences in these details in the "Source Engine" and "Pre-built SDK engine" sections which are under heading 3 "###" below

Reply:
Good catch! I think that might be too far down to mention it though because they need to run it in this section, so I can add the options in this section. Let me know if you think it doesn't flow well and we can revamp.